File: src/main/java/com/example/project/entities/AccessAddress.java
package com.example.project.entities;

import java.io.Serializable;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.DBRef;
import org.springframework.data.mongodb.core.mapping.Document;

import lombok.Data;


@Data
@Document(collection = "AccessAddress")
public class AccessAddress implements Serializable {
	@Id
	private int accessAddress_id;
	private String host;
	private int port;
	private String ap;
    private Sensor sensor;
	
}

File: project/angular_project/src/app/models/access-address.ts
import { Sensor } from "./sensor";


export class AccessAddress {
  accessAddress_id: number;
  host: string;
  port: number;
  ap: string;
  sensor: Sensor;

  constructor() {
	this.accessAddress_id = 0;
    this.host = "";
    this.port = 0;
    this.ap = "";
    this.sensor = new Sensor();
  }
}
